pages/apim/3.x/installation-guide/docker/installation-guide-docker-compose-quickstart-ee.adoc

```diff
@@ -0,0 +1,63 @@
+= Quick Install with Docker Compose  (Enterprise Edition)
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/apim_installation_guide_docker_compose_quickstart_ee.html
+:page-folder: apim/installation-guide/docker
+:page-layout: apim3x
+
+This page describes how to install and run APIM Enterprise Edition in Docker containers on `localhost` using the `docker compose` command.
+
+== Prerequisites
+
+Docker must be installed and running. For more information about installing Docker, see the link:{{'https://www.docker.com/' | relative_url }}[Docker website].
+
+You must have a license key for the APIM Enterprise Edition. For more information about getting a license key, visit the link:https://www.gravitee.io/pricing[Gravitee pricing page].
+
+== Installing APIM
+
+1. Download the `docker-compose.yml` file as `docker-compose-apim-ee.yml`.
++
+[code,bash]
+----
+curl https://raw.githubusercontent.com/gravitee-io/gravitee-docker/master/apim/3.x/docker-compose.yml -o docker-compose-apim-ee.yml  
+----
+
+2. Open `docker-compose-apim-ee.yml` in a text editor, and under `/services/gateway/volumes` add the following line.
++
+[code,yml]
+----
+ - /gravitee/license.key:/opt/graviteeio-gateway/license/license.key
+----
++
+Where `/gravitee/license.key` is the full path to the license key. This ensures that the gateway can access the license key.
```

Comment:
```suggestion
Where `/path/to/license.key` is the relative path from the directory containing the `docker-compose-apim-ee.yml` file to the license key. This ensures that the gateway can access the license key.
```

Reply:
I understand the point you are making, but I think it is simpler to use absolute paths here, especially as this is a one-off process.